CUDADataFormats/Track/interface/TrajectoryStateSoA_t.h

```diff
@@ -1,11 +1,11 @@
-#ifndef CUDADataFormatsTrackTrajectoryStateSOA_H
-#define CUDADataFormatsTrackTrajectoryStateSOA_H
+#ifndef CUDADataFormatsTrackTrajectoryStateSOA_t_H
+#define CUDADataFormatsTrackTrajectoryStateSOA_t_H
```

Comment:
If you rename the main struct/class in the file to `TrajectoryStateSoAT`, the file name be renamed to `TrajectoryStateSoAT.h`, and the include guards to
```suggestion
#ifndef CUDADataFormats_Track_TrajectoryStateSoAT_h
#define CUDADataFormats_Track_TrajectoryStateSoAT_h
```

Reply:
Done in the latest commits.